docs/reference/glossary-of-terms.md

```diff
@@ -0,0 +1,227 @@
+
+#### Knowledge Graph
+A network of real-world entities (i.e., objects, events, situations, and concepts) that illustrates the relationships between them. Knowledge graphs (in relation to ontologies) are thought of as real data built using an ontology as a framework.
+
+#### Label
+Usually refers to a human-readable label corresponding to the `rdfs:label` [predicate](#predicate). Labels are typically unique per ontology. In OBO Format and in the bio-ontology literature, labels are sometimes called [Names](#Name). Sometimes in the machine learning literature, and in databases such as Neo4J, "label" actually refers to a [Category](#Category).
+
+#### Lutra
+[Lutra](https://ottr.xyz/#Lutra) is the open source reference implementation of the OTTR templating language.
+
+#### Mapping
+See [SSSOM]('SSSOM')
+
+#### Materialised
+The process of calculating the implicit statements in an ontology - making inferred axioms explicit by asserting them.
+
+#### Name
+Usually synonymous with [Label](#Label), but in the formal logic and OWL community, "Name" sometimes denotes an [Identifier](#Identifier)
+
+#### Named Individual
+An [Individual](#Individual) that is given an explicit name that can be used in any ontology to refer to the same object.
+
+#### Nodes
+[Terms](#Term) represented in a [graph](#Graph)
+
+#### Object
+The "right" side of a [Triple](#Triple).
+
+#### ObjectProperty
+A simple association between two terms, also see [predicate](#predicate) & [Relationship](#Relationship). 
+
+#### OBO
+Open Biological and Biomedical Ontology. This could refer to the [OBO Foundry](https://obofoundry.org/) (e.g. OBO ontologies = ontologies that follow the standards of the OBO Foundry) or [OBO Format](#OBO-Format)
+
+#### OBO Format
+A serialization format for ontologies designed for easy viewing, direct editing, and readable diffs. It is popular in bioinformatics, but not widely used or known outside the genomics sphere. OBO is mapped to OWL, but only expresses a subset, and provides some OWL abstractions in a more easy to understand fashion.
+
+#### OLS
+Ontology Lookup Service. An [Ontology Repository](#Ontology-Repository) that is a curated collection of multiple biologically relevant ontologies, many from [OBO](#OBO). OLS can be accessed with this [link](https://www.ebi.ac.uk/ols/index)
+
+#### Ontology
+A flexible concept loosely encompassing any collection of [Ontology elements](#Ontology-element) and statements or relationships connecting them.
+
+#### ODK
+[Ontology Development Kit](https://github.com/INCATools/ontology-development-kit). A toolkit and docker image for managing ontologies.
+
+#### Ontology element
+A discrete part of an [Ontology](#Ontology), with a unique persistent identifier. The most important elements are [Terms](#Term) but other elements can include various metadata artefacts like [Annotation Properties](#Annotation-Property) or [Subsets](#Subset)
+
```

Comment:
Term is confusing here.  Elsewhere Ontology element seems to correspond to OWL Entity.  Might be better to make that clearer.  Also odd to see Subset here.